src/ontology/mondo-edit.obo

```diff
@@ -432964,6 +432967,23 @@ intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: realized_in_response_to_stimulus NCBITaxon:1335626 ! Middle East respiratory syndrome-related coronavirus
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100124
+name: NAA10-related syndrome
+def: "Any X-linked syndromic intellectual disability in which the cause of the disease is a mutation in the NAA10 gene. Patients with variants in the NAA10 gene demonstrate symptoms such as developmental delay, intellectual disability, autism spectrum disorder, hypotonia, facial dysmorphism, cardiac anomalies, and/or skeletal anomalies." [https://clinicalgenome.org/affiliation/40006/, PMID:23871722, PMID:24431331, PMID:25099252, PMID:26522270, PMID:27094817, PMID:28967461, PMID:29558889, PMID:29748569, PMID:31088393, PMID:31127942, PMID:31174490]
```

Comment:
Def should match name.

Reply:
I was following the pattern for the definition:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/disease_series_by_gene.yaml

The parent of this class is 'X-linked syndromic intellectual disability'

Should the definition read: "NAA10-related syndrome is an X-linked syndromic intellectual disability in which..."?

I will revise this.